interfaces/Portalicious/src/app/pages/project-registrations/components/change-status-reason/change-status-reason.component.html

```diff
@@ -0,0 +1,17 @@
+<div class="space-y-4">
+  <br />
+  <app-form-field-wrapper
+    label="Add a reason for {{ statusVerbProgressive() | lowercase }}"
+    i18n-label
+    [errorMessage]="reasonValidationErrorMessage() ?? ''"
+  >
+    <input
+      pInputText
+      id="reason"
+      [(ngModel)]="reason"
+      [class.invalid]="reasonValidationErrorMessage()"
```

Comment:
I've never seen this syntax before. I can't tell if it works because the styling of the input is broken, but if it does, very cool!

Reply:
It was not working :), so I changed it